### DISCUSSION TITLE
orchestrator bare-metal-ansible issues

### DISCUSSION THREAD

**Discussion by alaabouz** [magma/magma#14752] [Q&A]
Hello,

I am having issues while running the deploy.sh script, for now i have this erros and i am not sure how to solve it out !
it's while calling the orc8r/tasks/main.yml and i am getting the following
TASK [magma/orc8r : orc8r | run DB setup script] *********************************************************************************************************************************************************************************************
fatal: [compute1]: FAILED! => {"changed": true, "cmd": "kubectl -n mariadb exec -i mariadb-0 -- mysql -u root \"--password=*************\" mysql < db_setup.sql", "delta": "0:00:00.364331", "end": "2022-12-22 16:20:48.972051", "msg": "non-zero return code", "rc": 1, "start": "2022-12-22 16:20:48.607720", "stderr": "ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)\ncommand terminated with exit code 1", "stderr_lines": ["ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)", "command terminated with exit code 1"], "stdout": "", "stdout_lines": []}

not sure how to solve the access denied for the root user.

appreciate your support !
Alaa.

**Comment by bensternthal** [comment]
Can you try to reproduce with the latest version?
